src/main/kotlin/de/bund/digitalservice/useid/statics/GlobalStaticError.kt

```diff
@@ -12,36 +13,34 @@ import kotlin.text.Charsets.UTF_8
 @Component
 class GlobalStaticError(
     private val applicationProperties: ApplicationProperties,
-    private val globalStaticProperties: GlobalStaticProperties,
-    private val widgetProperties: WidgetProperties
+    private val messageSource: MessageSource
 ) : DefaultErrorAttributes() {
 
     override fun getErrorAttributes(request: ServerRequest, options: ErrorAttributeOptions): Map<String, Any> {
         val errorAttributes = super.getErrorAttributes(request, options)
         val statusCode = errorAttributes["status"] as Int
 
         val customGlobalErrorAttributes = mapOf(
-            "localizationError" to globalStaticProperties.errorView.localization,
+            "showReportEmail" to true,
             "errorReportEmailLink" to createEmailReportLink(statusCode),
-            "baseUrl" to applicationProperties.baseUrl,
-            "metaTag" to widgetProperties.metaTag
+            "baseUrl" to applicationProperties.baseUrl
         )
 
         return errorAttributes + customGlobalErrorAttributes
     }
 
     private fun createEmailReportLink(statusCode: Int): String {
-        val emailAddress = globalStaticProperties.errorView.localization.errorReportEmail
-        val errorReportSubject = globalStaticProperties.errorView.localization.errorReportSubject
-        val errorReportBody = globalStaticProperties.errorView.localization.errorReportBody
+        val emailAddress = messageSource.getMessage("error.default.report-email", null, LocaleContextHolder.getLocale())
```

Comment:
We could assign `LocaleContextHolder.getLocale()` to a variable:

`val currentLocale = LocaleContextHolder.getLocale()`

Reply:
will do